src/components/simple-water.js

```diff
@@ -287,7 +280,7 @@ AFRAME.registerComponent("simple-water", {
 
   init() {
     if (!waterNormalMap) {
-      waterNormalMap = new TextureLoader().load(waterNormalsUrl);
+      waterNormalMap = new HubsTextureLoader().load(waterNormalsUrl);
```

Comment:
Out of scope for this PR, but I wonder if we should just use a single instance of HubsTextureLoader for the whole app.